snap/snapcraft.yaml

```diff
@@ -151,6 +151,14 @@ parts:
       - jq
       - pciutils
 
+  # NOTE: the parts: engine, docker-cli, containerd, runc, tini, buildx
+  # and compose, had their tags versions set by installing the deb packages
+  # from docker archive (https://docs.docker.com/engine/install/ubuntu/)
+  # and extracting then with:
+  # `docker version`
+  # `docker buildx version`
+  # `docker compose version`
+
```

Comment:
I'd leave this a one-liner close to the part itself. This is large snapcraft file and it is hard to know if the documentation is far away from it matters.